src/collector/server/server.go

```diff
@@ -44,11 +44,7 @@ import (
 // RunOptions provides options for running the server
 // with backwards compatibility if only solely adding fields.
 type RunOptions struct {
-	// ConfigFile is the config file to use.
-	ConfigFile string
```

Comment:
N.B.: definitely backwards incompatible; are we expecting people to embed these? If they are, I can bring this back. cc @benraskin92